Documentation/PageTsconfig/Mod.rst

```diff
@@ -703,20 +703,20 @@ csvDelimiter
 ------------
 
 :aspect:`Datatype`
-    string
+   string
 
 :aspect:`Description`
-    Defines the delimiter for CSV exports (Microsoft Excel expects `;` to be set).
+   Defines the default delimiter for CSV exports (Microsoft Excel expects
+   `;` to be set). The value set will be displayed as default delimiter in the
+   export dialog in the list module.
```

Comment:
References here (and below) should point to the "download dialog" instead of "export"